informal_update/enums.py

```diff
@@ -0,0 +1,91 @@
+import enum
+from types import DynamicClassAttribute
+
+from django.utils.functional import Promise
+
+__all__ = ['Choices', 'IntegerChoices', 'TextChoices']
+
+
+class ChoicesMeta(enum.EnumMeta):
+    """A metaclass for creating a enum choices."""
+
+    def __new__(metacls, classname, bases, classdict, **kwds):
```

Comment:
Let's move this to main/ as well. Also, add a note stating that this is already implemented in the newer version of Django and need to use that instead of this only upgrade is done.